src/dlfcn.c

```diff
@@ -608,11 +642,16 @@ static void *get_address_from_import_address_table( void *iat, DWORD iat_size, v
 /* Holds module filename */
 static char module_filename[2*MAX_PATH];
 
-static BOOL fill_module_info( void *addr, Dl_info *info )
+static BOOL fill_module_info( HMODULE hModuleImport, void *addr, Dl_info *info )

```

Comment:
As this function does more than filling module info, I would name it fill_info() or similiar.

Reply:
done!